doc/create_php_binary.md

```diff
@@ -1,19 +1,29 @@
 # Creating PHP binary 
 
-1. Create a EC2 large instance
-2. Copy compile_php.sh to instance
+1. Create a EC2 large instance based on 
+[the current supported version AMI](https://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html).
+2. Connect to the instance after it boots to accept it to your known hosts:
 
-    From this project's root directory:
-    
         $ export AWS_IP=ec2-user@{ipaddress}
         $ export SSH_KEY_FILE=~/.ssh/aws-key.rsa
+        $ ssh -i $SSH_KEY_FILE $AWS_IP
```

Comment:
Why `ssh` into the box?

Reply:
The scp doesn’t work until you add the host to known_hosts, for me anyway. It just hangs. Once I ssh’d in and accepted the entry the scp command worked fine.

Reply:
ah